workloads/kube-burner/alerts-profiles/cluster-density.yml

```diff
@@ -0,0 +1,43 @@
+# etcd
```

Comment:
I agree in general that etdc should respond w/in these time frames but in azure the disks are slower to the point where the thresholds have been adjusted so etcd doesnt ping-pong around nodes. Not sure if we want to have some logic to deal w/ Azure

Reply:
Per our slack convo. An Azure profile will be added in a separate PR.